apps/etterlatte-beregning/src/main/kotlin/avkorting/Avkorting.kt

```diff
@@ -151,31 +151,48 @@ data class Avkorting(
         return oppdatertMedNyInntekt.beregnAvkortingRevurdering(beregning, sanksjoner)
     }
 
+    private fun haandterOpphoerDato(
+        opphoerFom: YearMonth? = null,
+        nyttGrunnlag: AvkortingGrunnlagLagreDto,
+    ): YearMonth? =
+        opphoerFom?.let {
+            if (it.year > nyttGrunnlag.fom.year) {
+                null
```

Comment:
Dette fungerer men jeg tror det er nyttig å sette tom desember. Hvis ikke vil det i visningen i Gjenny har to åpne peridoer (tror jeg).

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/6282/commits/747602316d94cdb4d9514c40a4583035aabb6c3b